scaleway/helpers_lb.go

```diff
@@ -207,3 +207,29 @@ func expandLbHCHTTPS(raw interface{}) *lb.HealthCheckHTTPSConfig {
 		Code:   expandInt32Ptr(rawMap["code"]),
 	}
 }
+
+func expandLbCertificateLetsEncrypt(raw interface{}) *lb.CreateCertificateRequestLetsencryptConfig {
+	if raw == nil || len(raw.([]interface{})) != 1 {
+		return nil
+	}
+	rawMap := raw.([]interface{})[0].(map[string]interface{})
+	alternativeNames := rawMap["subject_alternative_name"].([]interface{})
+	config := &lb.CreateCertificateRequestLetsencryptConfig{
+		CommonName:             rawMap["common_name"].(string),
+		SubjectAlternativeName: make([]string, 0, len(alternativeNames)),
```

Comment:
Can you remove this line please, `append` can handle nil slices.
It adds complexity and no performance optimization is required here.